projects/smart-table/src/lib/smart-table/smart-table.service.ts

```diff
@@ -2,10 +2,15 @@ import { Injectable } from '@angular/core';
 import { HttpHeaders } from '@angular/common/http';
 import { HttpClient } from '@angular/common/http';
 import { Observable } from 'rxjs';
-import { map } from 'rxjs/operators';
+import * as queryString from 'query-string';
+import * as FileSaver from 'file-saver';
+import * as XLSX from 'xlsx';
 
 import { SmartTableDataQuery } from './smart-table.types';
 
+const EXCEL_EXTENSION = '.xlsx';
```

Comment:
Je mag deze opmerking negeren als je wilt, maar zou het niet beter zijn dat we hier de meer universele CSV extensie gebruiken?

Reply:
Kunnen het ook configureerbaar maken.